code/modules/antagonists/gang/gang.dm

```diff
@@ -81,69 +95,93 @@
-	package_spawner.my_gang_datum = src
+	if(starter_gangster)
+		package_spawner.Grant(owner.current)
+		package_spawner.my_gang_datum = src
 	var/mob/living/M = mob_override || owner.current
 	add_antag_hud(antag_hud_type, antag_hud_name, M)
-	if(M.hud_used)
-		var/datum/hud/H = M.hud_used
-		var/atom/movable/screen/wanted/giving_wanted_lvl = new /atom/movable/screen/wanted()
-		H.wanted_lvl = giving_wanted_lvl
-		giving_wanted_lvl.hud = H
-		H.infodisplay += giving_wanted_lvl
-		H.mymob.client.screen += giving_wanted_lvl
 
 /datum/antagonist/gang/remove_innate_effects(mob/living/mob_override)
-	package_spawner.Remove(owner.current)
+	if(starter_gangster)
+		package_spawner.Remove(owner.current)
 	var/mob/living/M = mob_override || owner.current
 	remove_antag_hud(antag_hud_type, M)
-	if(M.hud_used)
-		var/datum/hud/H = M.hud_used
-		H.infodisplay -= H.wanted_lvl
-		QDEL_NULL(H.wanted_lvl)
 	..()
 
-/// Passes points gained to the family team.
-/datum/antagonist/gang/proc/add_gang_points(points_to_add)
-	if(my_gang)
-		my_gang.adjust_points(points_to_add)
-
-/// Determines if a gang has completed their special objective.
-/datum/antagonist/gang/proc/check_gang_objective()
-	return TRUE
-
 /// Gives a gangster their equipment in their backpack and / or pockets.
 /datum/antagonist/gang/proc/equip_gangster_in_inventory()
 	if(istype(owner.current, /mob/living/carbon/human))
-		for(var/C in my_gang.free_clothes)
-			var/obj/O = new C(owner.current)
-			var/list/slots = list (
-				"backpack" = ITEM_SLOT_BACKPACK,
-				"left pocket" = ITEM_SLOT_LPOCKET,
-				"right pocket" = ITEM_SLOT_RPOCKET
-			)
-			var/mob/living/carbon/human/H = owner.current
-			var/equipped = H.equip_in_one_of_slots(O, slots)
+		var/list/slots = list (
+			"head" = ITEM_SLOT_HEAD,
+			"suit" = ITEM_SLOT_OCLOTHING,
+			"jumpsuit" = ITEM_SLOT_ICLOTHING,
+			"shoes" = ITEM_SLOT_FEET,
+			"neck" = ITEM_SLOT_NECK,
+			"glasses" = ITEM_SLOT_EYES,
+			"gloves" = ITEM_SLOT_GLOVES,
+			"back" = ITEM_SLOT_BACK,
+			"mask" = ITEM_SLOT_MASK,
+			"belt" = ITEM_SLOT_BELT,
+			"ears" = ITEM_SLOT_EARS,
+			"backpack" = ITEM_SLOT_BACKPACK,
+			"left pocket" = ITEM_SLOT_LPOCKET,
+			"right pocket" = ITEM_SLOT_RPOCKET,
+			"hands" = ITEM_SLOT_HANDS
+		)
+		var/obj/item/gangster_cellphone/phone = new()
+		phone.gang_id = gang_name
+		phone.name = "[gang_name] branded cell phone"
+		var/mob/living/carbon/human/gangster_human = owner.current
+		var/phone_equipped = gangster_human.equip_in_one_of_slots(phone, slots, qdel_on_fail = FALSE, force_equip = !isplasmaman(gangster_human))
+		if(!phone_equipped)
+			to_chat(owner.current, "Your [phone.name] has been placed at your feet.")
+			phone.forceMove(get_turf(gangster_human))
+		for(var/clothing in my_gang.free_clothes)
+			var/obj/clothing_object = new clothing(owner.current)
+			var/equipped = gangster_human.equip_in_one_of_slots(clothing_object, slots, qdel_on_fail = FALSE, force_equip = !isplasmaman(gangster_human))
```

Comment:
If you're not going to use the `equipped` variable, and if you're going to list every slot anyway, why not remove that entire list and just use `equip_to_best_slot`?

Reply:
Here and everywhere else.